README.md

```diff
@@ -6,6 +6,7 @@
 [![Nightlies](https://github.com/containerd/containerd/workflows/Nightly/badge.svg)](https://github.com/containerd/containerd/actions?query=workflow%3ANightly)
```

Comment:
Can we also update the `Build Status` badge to https://github.com/containerd/containerd/actions/workflows/ci.yml/badge.svg?event=merge_group, as CI is now run on `merge_group` events. The above link will bring the badge back to green.